src/cmd/command.rs

```diff
@@ -32,6 +32,19 @@ pub enum CommandError {
     Killed(String),
 }
 
+impl CommandError {
```

Comment:
thisError already implement Display trait.
is there a need to add a to_string() ?

Reply:
You are right, here's a MR to change that => https://github.com/Qovery/engine/pull/640